pkgs/by-name/ch/chirpstack/package.nix

```diff
@@ -0,0 +1,258 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  yarnConfigHook,
+  yarnBuildHook,
+  fetchYarnDeps,
+  rustPlatform,
+  nix-update-script,
+  pnpm,
+  typescript,
+  nodejs,
+  protobuf,
+  protoc-gen-js,
+  protoc-gen-grpc-web,
+  gnumake,
+  cmake,
+  rdkafka,
+}:
+let
+  version = "4.10.1";
+
+  src = fetchFromGitHub {
+    owner = "chirpstack";
+    repo = "chirpstack";
+    rev = "v${version}";
+    hash = "sha256-0fllOtpiZ1uLPlM5Wes7HkOLvnDpQIUFkoY3Dr4KZ44=";
+  };
+
+  grpc-web = stdenv.mkDerivation (finalAttrs: {
+    pname = "chirpstack-grpc-web";
+    inherit version;
+    src = "${src}/api/grpc-web";
+
+    offlineCache = fetchYarnDeps {
+      yarnLock = "${finalAttrs.src}/yarn.lock";
+      hash = "sha256-T2+N2U0QAVXQBXIvSgpOpO/XLNyAGmvH67VQbTkm2kk=";
+    };
+
+    buildPhase = ''
+      runPhase preBuild
+
+      substituteInPlace Makefile \
+      --replace-fail "../proto" "${src}/api/proto" \
```

Comment:
```suggestion
      --replace-fail "../proto" "${finalAttrs.src}/api/proto" \
```

Reply:
This is actually intended here as the Protobuf definitions for `api/grpc-web` are located in `api/proto`.